op-program/host/sources/sources.go

```diff
@@ -0,0 +1,27 @@
+package sources
```

Comment:
For reviewers: New package due to cyclical imports between prefetcher <-> config in the `host`.

Reply:
This makes me even more determined to get this out of config. :)